.aegir.js

```diff
@@ -2,7 +2,8 @@
 
 const createServer = require('./src').createServer
 
-const server = createServer()
+const server = createServer({ port: 25000 })
```

Comment:
Can we use port 0 to avoid port collisions?

Reply:
This is spawned in a separate process by aegir, we would have to create _yet another side channel_ just to pass this port over. Not worth it. For other cases, yes I like port 0 too.